transformers/integration_tests/vow_file_auction_attributes_test.go

```diff
@@ -37,23 +37,23 @@ var _ = Describe("VowFile LogNoteTransformer", func() {
 
 	BeforeEach(func() {
 		test_config.CleanTestDB(db)
-		vowFileConfig := event.TransformerConfig{
-			TransformerName:   constants.VowFileTable,
+		vowFileAuctionAttributesConfig := event.TransformerConfig{
+			TransformerName:   constants.VowFileAuctionAttributesTable,
 			ContractAddresses: []string{test_data.VowAddress()},
 			ContractAbi:       constants.VowABI(),
-			Topic:             constants.VowFileSignature(),
+			Topic:             constants.VowFileAuctionAttributesSignature(),
 		}
 
-		addresses = event.HexStringsToAddresses(vowFileConfig.ContractAddresses)
-		topics = []common.Hash{common.HexToHash(vowFileConfig.Topic)}
+		addresses = event.HexStringsToAddresses(vowFileAuctionAttributesConfig.ContractAddresses)
+		topics = []common.Hash{common.HexToHash(vowFileAuctionAttributesConfig.Topic)}
 
 		initializer = event.ConfiguredTransformer{
-			Config:      vowFileConfig,
-			Transformer: vow_file.Transformer{},
+			Config:      vowFileAuctionAttributesConfig,
+			Transformer: auction_attributes.Transformer{},
 		}
 	})
 
-	It("fetches and transforms a Vow.file event", func() {
+	It("fetches and transforms a Vow.file auction attributes event", func() {
```

Comment:
You could check for a msgSender value as well here too. I don't think it's consistent, but i know many tests are checking.